pkg/plugins/backendplugin/manager/plugin_settings_test.go

```diff
@@ -41,5 +42,55 @@ func TestPluginSettings(t *testing.T) {
 			require.Len(t, env, 3)
 			require.EqualValues(t, []string{"GF_PLUGIN_KEY1=value1", "GF_PLUGIN_KEY2=value2", "GF_VERSION=6.7.0"}, env)
 		})
+		t.Run("Should override config variable with environmental variable ", func(t *testing.T) {
```

Comment:
Nit, but better to have blank lines between `t.Run` invocations for readability.
```suggestion

		t.Run("Should override config variable with environmental variable ", func(t *testing.T) {
```